x-pack/plugins/enterprise_search/public/applications/app_search/components/layout/kibana_header_actions.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiButtonEmpty, EuiText, EuiFlexGroup, EuiFlexItem, EuiHeaderLinks } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export const KibanaHeaderActions: React.FC = () => {
+  return (
+    <EuiHeaderLinks>
+      <EuiFlexGroup gutterSize="s">
+        <EuiFlexItem>
+          <EuiButtonEmpty iconType="beaker">
+            <EuiText size="s">
```

Comment:
An EuiText size of "s" within a button seems odd / like it might not work as expected - are you sure you don't want this instead?

```suggestion
          <EuiButtonEmpty iconType="beaker" size="s">
```
 Or alternatively, removing `size="s"` totally?

Reply:
Ah, totally happy to try that out. I just copied this from WS.

Reply:
027792df5d7809758e7149a9aa60d67ce06e31d0

Reply:
@scottybollinger @yakhinvadim FYI, I updated the WS kibana_header_actions file as well for consistency.

Reply:
Nice! Thanks for cleaning this up for the WS folks ✨